repository/l.json

```diff
@@ -1559,6 +1559,18 @@
 				}
 			]
 		},
+		{
+			"name": "LogbackJsonLogViewer",
+			"author": ["neno--"],
```

Comment:
Please remove the “author” field, it’s not needed since it’s just your username.

Reply:
Sorry for not catching that earlier!